zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedOrErroredRuntimeChecker.scala

```diff
@@ -45,13 +45,37 @@ object DeletedOrErroredRuntimeChecker {
           res <- runtimeOpt match {
             case None =>
               (if (isDryRun) F.unit
-               else
-                 dbReader.markRuntimeDeleted(runtime.id)).as(Some(runtime))
-            case Some(instance) =>
-              if (instance.getStatus.getState == com.google.cloud.dataproc.v1.ClusterStatus.State.ERROR) {
+               else {
+                 for {
+                   _ <- dbReader.markRuntimeDeleted(runtime.id)
+                   _ <- dbReader.insertClusterError(
```

Comment:
Why are we inserting an error record for a cluster that was deleted?

Reply:
that was my original thought as well..But I think it does make sense to have the error message there because this way, user will see this error message, as opposed to no clue what happened

Reply:
I don't think the user sees Deleted runtimes, but I feel like the information is still good to have in the DB -- e.g. that a cluster was deleted via zombie monitor rather than the user.